Mathlib/RingTheory/DedekindDomain/FiniteAdeleRing.lean

```diff
@@ -336,9 +336,30 @@ instance : Algebra R (FiniteAdeleRing R K) :=
 instance : IsScalarTower R K (FiniteAdeleRing R K) :=
   IsScalarTower.of_algebraMap_eq' rfl
 
+@[coe]
+def coe (x : FiniteAdeleRing R K) : K_hat R K := x.1
```

Comment:
Can you add a docstring to your definitions? thats why CI is failing.

Reply:
Added.